external-crates/move/move-stdlib/sources/type_name.move

```diff
@@ -13,9 +13,20 @@ module std::type_name {
         name: String
     }
 
-    /// Return a value representation of the type `T`.
+    /// Return a value representation of the type `T`.  Package IDs
+    /// that appear in fully qualified type names in the output from
+    /// this function are original IDs (the ID of the first version of
+    /// the package, even if the type in question was introduced in a
+    /// later upgrade).
```

Comment:
I think this comment is off, just a bad copy paste?

Reply:
Ah yes, thanks for spotting -- this has already landed so I'll fix in a follow-up.